CMakeLists.txt

```diff
@@ -125,12 +122,6 @@ else()
   endif()
 endif()
 
-if(KAHYPAR_USE_COMPATIBLE_TBB_VERSION)
```

Comment:
This was actually a nice feature, that we may want to keep around?

Reply:
could be an idea for environment where you not allowed to install something